src/main/java/net/mcreator/ui/MCreator.java

```diff
@@ -382,9 +382,9 @@ public boolean closeThisMCreator(boolean returnToProjectSelector) {
 			}
 
 			application.getDiscordClient()
-					.updatePresence("Working on " + workspace.getWorkspaceSettings().getModName() + tabAddition,
-							Launcher.version.getMajorString() + " for " + workspace.getGenerator()
-									.getGeneratorMinecraftVersion(),
+					.updatePresence(L10N.t("dialog.discord_rpc.working_on", workspace.getWorkspaceSettings().getModName() + tabAddition),

```

Comment:
Do not externalize these messages, Discord messages should always be English

Reply:
Sorry